pkg/providers/vsphere/config/template.yaml

```diff
@@ -270,7 +278,11 @@ spec:
     - content: |
 {{ .auditPolicy | indent 8 }}
       owner: root:root
+{{- if (eq .format "bottlerocket") }}
```

Comment:
I think you can leave this one as /etc as well since we have the symlink being created in the BR bootstrap code for `/etc/kubernetes -> /var/lib/kubeadm`.  As is, this actually wont work as you expect since /var/lib on the bootstrap container is not actually the host /var/lib.  BR mounts the host folder at `/.bottlerocket/rootfs` so the symlink is actually `/etc/kubernetes -> /.bottlerocket/rootfs/var/lib`

Reply:
If I leave this at `/etc/kubernetes` will I also need to do [this patch](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/kubernetes-sigs/cluster-api/patches/0004-Generating-proper-vip-destnation-path-for-bottlerock.patch#L110-L136)?

Reply:
That patch is doing the hostPath translation from /etc/kubernetes to /var/lib/kubeadm.  The way you have this code with the hostPath being /var/lib for in the BR *should* just work as is, no patch to capi needed.  Again, personally I think we should patch in capi to do this translation more generally like we are doing kube-vip in the patch you referenced to avoid peppering this logic through various repos.

cc @vignesh-goutham @g-gaston

Reply:
I take back what i said. I realize now we are doing the same thing in this file for the controller-manager/scheduler mountPaths.  We added this in addition the capi patch to support the upgrade use case, I forget all  the details but it had something to do with the configmap capi uses to setup the new nodes during upgrade.  I think for now this is should be fine, but we should consider a way to move all these if BR checks related to paths out to a more central place in capi or cli.

Reply:
Thanks for the insight @jaxesn ! We can merge this for now and address cleaning the template up as a separate task.